srcpkgs/ruby-travis/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'ruby-travis'
+pkgname=ruby-travis
+version=1.8.9
+revision=1
+noarch=yes
+build_style=gem
+depends="ruby-backports ruby-faraday>=0.9 ruby-faraday_middleware>=0.9
+ ruby-faraday_middleware>=0.9.1 ruby-gh>=0.13 ruby-highline>=1.6
+ ruby-launchy>=2.1 ruby-pusher-client>=0.4 ruby-typhoeus>=0.6
+ ruby-typhoeus>=0.6.8"
```

Comment:
Seems to me as if ruby-typhoeus is specified twice

Reply:
same as above